src/main/resources/application.conf

```diff
@@ -2,8 +2,8 @@ stripe-scala {
   endpoint="https://api.stripe.com"
   endpoint=${?STRIPE_ENDPOINT}
 
-  file-upload-endpoint="https://uploads.stripe.com"
-  file-upload-endpoint=${?STRIPE_FILE_UPLOAD_ENDPOINT}
+  fileUploadEndpoint="https://uploads.stripe.com"
```

Comment:
I was under the impression that the convention Typesafe config was to use hyphens:

See the following examples:
- https://github.com/typesafehub/config
- https://github.com/akka/akka/blob/master/akka-remote/src/main/resources/reference.conf